anubis/entrypoints/puppet-lint

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+# Establish source dir for relative includes
+DIR="${BASH_SOURCE%/*}"
+if [[ ! -d "$DIR" ]]; then DIR="$PWD"; fi
+
+# Pre-suite (download and extract requested release)
+# shellcheck source=../shared/entrypoint-pre disable=SC1091
+. "$DIR/../shared/entrypoint-pre"
+
+# Make sure required Anubis env vars are set
+# shellcheck source=../shared/entrypoint-anubis-pre disable=SC1091
+. "$DIR/../shared/entrypoint-anubis-pre"
+
+# TODO make a PDK frontend/backdoor for this? like a wrapper in /opt/puppetlabs/pdk/private?
+export PATH=$PATH:/opt/puppetlabs/pdk/private/ruby/2.5.8/bin:/opt/puppetlabs/pdk/private/ruby/2.5.8/lib/ruby/gems/2.5.0
+export GEM_PATH=/opt/puppetlabs/pdk/private/ruby/2.5.8/lib/ruby/gems/2.5.0:/opt/puppetlabs/pdk/share/cache/ruby/2.5.0:/opt/puppetlabs/pdk/private/puppet/ruby/2.5.0
```

Comment:
I'm not totally sure why this required ruby `2.5.8` rather than `2.5.7` like `strings`. Both entrypoints seemed to work running locally.

Reply:
Hmmm, I would suspect that we would need to update both to 2.5.8 since that is what is shipped in puppet-runtime now... are you sure that 2.5.7 works with the latest `pdk` docker image?

Reply:
So I don't think `2.5.7` is present, but `strings` seems to work anyways. This is running locally, and it ran successfully both from my local API instance as well as from within the container.

Reply:
Hmmm, yeah maybe it just doesn't need anything from that part of the ruby `GEM_PATH`, I think it wouldn't hurt to update the strings one to 2.5.8 though.

Reply:
That sounds good. I added a commit for that, and after rebuilding the image and running locally, looks like it's still working.